lib/nerves_hub/workers/schedule_org_audit_log_truncation.ex

```diff
@@ -0,0 +1,42 @@
+defmodule NervesHub.Workers.ScheduleOrgAuditLogTruncation do
+  use Oban.Worker,
+    max_attempts: 5,
+    queue: :truncation
+
+  alias NervesHub.Accounts
+  alias NervesHub.Workers.OrgAuditLogTruncation
+
+  @impl true
+  def perform(_) do
+    if enabled?() do
+      orgs = Accounts.get_orgs()
+
+      Enum.each(orgs, fn org ->
+        {:ok, _} =
+          org
+          |> truncation_args()
+          |> OrgAuditLogTruncation.new()
+          |> Oban.insert()
+      end)
+
+      {:ok, Enum.count(orgs)}
+    else
+      {:ok, :disabled}
+    end
+  end
+
+  defp truncation_args(org) do
+    %{
+      org_id: org.id,
+      days_to_keep: days_to_keep(org)
+    }
+  end
+
+  def days_to_keep(org) do
+    org.audit_log_days_to_keep || config()[:default_days_kept]
+  end
+
+  defp enabled?(), do: config()[:enabled]
```

Comment:
```suggestion
  defp enabled?(), do: config()[:enabled] == "true"
```

If the config value is `"false"` then this would still evaluate to run

Reply:
in `runtime.exs`
```
config :nerves_hub, :audit_logs,
  enabled: System.get_env("TRUNATE_AUDIT_LOGS_ENABLED", "false") == "true",
```
so we should always have a boolean